### PR TITLE
update java/go latest with variant version

### PR DESCRIPTION
**Pull request by strongjz**

### CODE REVIEW

**Review by amouat — APPROVED** [review]

File: images/go/image.yaml [hunk 1]
```diff
@@ -13,11 +13,11 @@ versions:
       config: configs/1.19-dev.apko.yaml
       extractTagsFrom:
         package: go-1.19
-        prefix: dev-
         exclude:
           - 1
   - apko:
       config: configs/latest-dev.apko.yaml
       extractTagsFrom:
-        prefix: dev-
```
Comment (jdolitsky) [inline images/go/image.yaml:22]: without a prefix, this will overwrite the latest one. we can just remove `extractTagsFrom:` altogether

**Review by jdolitsky — CHANGES_REQUESTED** [review]

File: images/go/image.yaml [hunk 2]
```diff
@@ -13,11 +13,11 @@ versions:
...
         package: go-1.20
+      tags:
+        - go-1.20-dev
```
Comment (jdolitsky) [inline images/go/image.yaml:23]: why have a tag with "go-" in the name? this is the go image. Are you looking for a tag suffix basically?

**Review by jdolitsky — CHANGES_REQUESTED** [review]

File: images/go/image.yaml (hunk 2 above)
Comment (strongjz) [reply images/go/image.yaml:23]: good catch, ill remove the go-

File: images/go/image.yaml (hunk 1 above)
Comment (strongjz) [reply images/go/image.yaml:22]: fixed

File: images/jdk/image.yaml [hunk 3]
```diff
@@ -14,11 +14,12 @@ versions:
   - apko:
       config: configs/latest-dev.apko.yaml
       extractTagsFrom:
-        package: openjdk-11
+        package: openjdk-17
         prefix: openjdk-
```
Comment (jdolitsky) [inline images/jdk/image.yaml:18]: like the other comment, this is going to overwrite latest above. should just remove `extractTagsFrom`. This was already broken before your changes it seems

**Review by jdolitsky — CHANGES_REQUESTED** [review]

File: images/jdk/image.yaml (hunk 3 above)
Comment (strongjz) [reply images/jdk/image.yaml]: updated

**Review by jdolitsky — APPROVED** [review]

**Pull request merged**